packages/web-twig/src/Resources/components/Tooltip/README.md

```diff
@@ -72,10 +72,89 @@ These attributes will be passed to the topmost HTML element of the component.
 
 ### TooltipWrapper
 
+⚠️ `TooltipWrapper` component is [deprecated] and will be renamed to the `Tooltip` in the next major version.
```

Comment:
I think there should be `tooltip` deprecation too?

Reply:
deprecation added